nvfuser/contrib/nn/normalization.py

```diff
@@ -113,14 +113,11 @@ def norm_fusion_forward(
     is_spatial_dim = [True] * num_dims
     is_spatial_or_batch_dim = [True] * num_dims
 
-    num_stats = fd.define_scalar(1)
```

Comment:
wait. this doesn't look right....

why are we not using num_stats?

Reply:
wasn't num_stats used when momentum is missing?!?! Looks like a functional mismatch here.

Reply:
https://github.com/pytorch/pytorch/blob/8191fb3e060f600090c839d6b86c28dbb687b679/torch/nn/modules/batchnorm.py#L157

Reply:
I've no idea... I attempted to test `fd.ops.batch_norm` and realized the logic here isn't the easiest to follow :) I may give up and create a new test.